src/server/editorServices.ts

```diff
@@ -360,7 +339,7 @@ namespace ts.server {
         /**
          * projects specified by a tsconfig.json file

```

Comment:
Actually it isnt same key..  Added comment to mapOfConfigFilePresence as:
```TypeScript
/**
         * This is a map of config file paths existance that doesnt need query to disk
         * - The entry can be present because there is inferred project that needs to watch addition of config file to folder
         *   In this case the exists could be true/false based on config file is present or not
         * - Or it is present if we have configured project open with config file at that location
         *   In this case the exists property is always true
         */
```